src/components/InvitePopover.tsx

```diff
@@ -77,7 +77,7 @@ export function InvitePopover(props: Props): React.ReactElement {
   return (
     <>
       <IconButton onClick={handleClick}>
-        <PersonAdd />
+        <Edit />
       </IconButton>
       <Popover
```

Comment:
If we make this a dialog rather than a popover, can we make it match Josh's design?

Reply:
Will turn the invite popover into a dialog in the next PR.